src/DataSource.ts

```diff
@@ -114,8 +117,18 @@ export class AlertmanagerDataSource extends DataSourceApi<CustomQuery, GenericOp
   }
 
   parseAlertAttributes(alert: any, fields: any[]): string[] {
-    const row: string[] = [alert.startsAt];
-    fields.slice(1).forEach((element: any) => {
+    var severityValue = 4;
+    if (alert.labels['severity'] === 'critical') {
```

Comment:
The severity values should be customizable, based on a variable defined globally in the datasource configuration panel or locally for each request.

Shouldn't you use a switch statement here instead of multiple `else if`?